services/streammanager/eventbridge/eventBridgeManager.go

```diff
@@ -0,0 +1,89 @@
+package eventbridge
```

Comment:
Let's keep the file name as per convention of the repo? `eventbridgemanager.go`?

Reply:
done